src/main/java/frc/robot/leds/LedStrip.java

```diff
@@ -0,0 +1,192 @@
+package frc.robot.leds;
+
+import edu.wpi.first.math.geometry.Translation3d;
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import java.awt.*;
```

Comment:
Probably not relevant

Reply:
What are you talking about?
All of the imports here are used.

Reply:
Including `java.awt`? That's the line I commented on

Reply:
That's the import of the `Color` class, but I will make it specifically `Color` and not a star import.

Reply:
Oh, I see. Why not use WPILib's [`Color`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/util/Color.html) class tho? That's the class WPILib's LEDs code uses, and it also has a nice amount of preset colors.

Reply:
Java's `Color` class has way more methods, options and preset colors.

Reply:
Sure, but do you need a wide variety of methods here? Do you use something the WPILib's class doesn't provide?
I don't want to force you to use WPILib's class, just want _you_ to make sure it's not enough.